packages/shared/src/utils/helpers/format-response.ts

```diff
@@ -42,6 +43,33 @@ export const formatPortfolioPosition = (
     };
 };
 
+export type TIDVErrorStatus = typeof idv_error_statuses[keyof typeof idv_error_statuses];
+
+//formatIDVError is parsing errors messages from BE (strings) and returns error codes for using it on FE
+export const formatIDVError = (errors: string[], status_code: string) => {
+    if (errors.length === 0 && status_code === 'none') return null;
+    const error_keys: Record<string, TIDVErrorStatus> = {
+        name: 'POI_NAME_MISMATCH',
+        birth: 'POI_DOB_MISMATCH',
+        rejected: 'POI_FAILED',
+    };
+    if (status_code === 'expired') {
```

Comment:
Please don't use hard-coded values in the code.
`identity_status_codes.expired`

Reply:
`identity_status_codes` is not part of shared package. It is part of account package.